packages/editor-ui/src/components/MainHeader/ExecutionDetails/ExecutionDetails.vue

```diff
@@ -36,6 +36,15 @@
 			</span>
 			{{ $locale.baseText('executionDetails.workflow') }}
 		</span>
+		<span
+			class="retry-exec-button"
+			v-if="!executionFinished && !executionWating && !successfulRetry">
```

Comment:
@Jhalter5Stones there seems to be a typo in `executionWating`. Please fix.

Reply:
@jspizziri Good catch, fixed.

Reply:
I'd go ahead and squash that.